deployment/main.tf

```diff
@@ -6,7 +6,7 @@ resource "openstack_compute_instance_v2" "dashboard" {
   network {
     uuid = var.net_id
   }
-  security_groups = [openstack_compute_secgroup_v2.secgroup.name, "default"]
+  security_groups = ["default", "HTTP", "motley-cue"]
```

Comment:
should we remove `default`?

Reply:
`ssh` is there, should I add it to `motley-cue` then?

Reply:
yes, I think it's better to have explicit control of the ports than to rely on groups that we don't control

Reply:
done, thanks